components/06-examples/samenwerken.hbs

```diff
@@ -0,0 +1,43 @@
+{{> '@header' }}
+<main>
+  <div class="row">
+    <div class="col-md-12">
+      <h1>Samenwerken aan behandelen</h1>
+      <h2 class="dso-steps-indicator">
+        Overzicht samenwerkingen
+      </h2>    
+    </div>
+    <div class="col-md-3">{{> '@button' label='samenwerking toevoegen' type='button' modifier='link' icon='icon fas fa-plus'}}
+    </div> 
+    <div class="col-md-12">
+      <table class="table">
+        <caption class="sr-only">Table met alle samenwerkingen</caption>
+        <thead>
+          <tr>
+            <th scope="col">Naam samenwerking</th>
+            <th scope="col">Status</th>
+            <th scope="col">Initiator</th>
+            <th scope="col">Verzoeknummer</th>
+            <th scope="col">Vertrouwelijkheid</th>
+            <th></th>
+            <th></th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td><a href="#fabien" ><b>Aanleg rondweg Leiden</b></a></td>
+            <td>Open</td>
+            <td>Gemeente Den Haag</td>
+            <td>123456789123</td>
+            <td>Geheim</td>
+            <td><a><span class="icon fas fa-users"></span></a></td>
```

Comment:
>As far as semantics and styling is concerned, the <a> element isn't a link (:link) unless it has an [href] attribute. A side-effect of this is that an <a> element without [href] won't be in the tabbing order by default.

https://stackoverflow.com/a/10510353

Ik denk dat je wat met die anchors wilt doen. Daarbij zijn ze nu niet toegankelijk, de anchors hebben geen naam. Ik zou voor het volgende gaan:
```
<a href="#">
  <span class="icon fas fa-users" aria-hidden="true"></span>
  <span class="sr-only">Gebruikers</span>
</a>
```

Reply:
Suggestie overgenomen en toegepast.

Reply:
Over het menu kan ik nog zeggen dat ik er flink mee bezig ben geweest om de context werkende te krijgen. Dus de menu's zijn nu ook een accurate weergave van de website zelf. 

In eerdere versies van de toolkit werkte de items in de context files, consistent niet bij mij.